docs/learn/learn-guides-polkadot-opengov.md

```diff
@@ -0,0 +1,64 @@
+---
+id: learn-guides-polkadot-opengov
+title: Polkadot OpenGov How-to Guides
+sidebar_label: OpenGov
+description: Advanced How-to Guides about Polkadot OpenGov.
+keywords: [opengov, polkadot opengov, referenda, cancel]
+slug: ../learn-guides-polkadot-opengov
+---
+
+import RPC from "./../../components/RPC-Connection";
+
+## Canceling Referenda using the Polkadot-JS UI
+
+:::info
+
+Anybody can cancel a referendum, given the referendum is ongoing (i.e., anywhere between Lead-in and
+voting/confirmation period). For more information about referenda timeline in Polkadot OpenGov see
```

Comment:
"ongoing" is any time until the confirmation period is over. So it's inclusive of "lead in", "deciding", and "confirming". I would be sure to say that it's inclusive; in English, it's ambiguous when you say "between" if it means including the initial and ending points or not.